analog-commander/usr/share/asteroid-launcher/watchfaces/analog-commander.qml

```diff
@@ -0,0 +1,499 @@
+                   : (index%5)==0 ? 0.6
+                   : displayAmbient ? 0.2
+                   : 0.5
+            width: index % 5 ? parent.width * 0.0050 : parent.width * 0.018
+
+            visible: ([0, 30].includes(index)) ? 0
+                   : 1
+
+            height: ([15, 45].includes(index)) ? parent.height * 0.060
+                  : index % 5 ? parent.height * 0.026
+                  : parent.height * 0.105
+
+            transform: Rotation {
+                origin.x: width / 2
+                origin.y: height / 2
+                angle: (index) * 6
+            }
+        }
+    }
+
+    Repeater {
+        model: 12
+        id: secondDots
+        Rectangle {
+            z: 0
+            antialiasing : true
+            property real rotM: (index + 2.5) / 12 
+            property real centerX: root.width / 2 - width / 2
+            property real centerY: root.height / 2 - height / 2
+            x: centerX+Math.cos(rotM * 2 * Math.PI) * parent.width * 0.463
+            y: centerY+Math.sin(rotM * 2 * Math.PI) * parent.width * 0.463
+            color: "#ff949494"
+	        opacity: displayAmbient ? 0.2
+		    	   : 0.5
+	        radius: 60
+            width: parent.width * 0.022
+            height: parent.height * 0.022
+            transform: Rotation {
+                origin.x: width / 2
+                origin.y: height / 2
+                angle: (index) * 6
+            }
+        }
+    }
+
+
+Repeater {
+        id: hourNumbers
+        model: 4
+        Text {
+            font {
+                family: "Teko"
+                styleName: "Regular"
+                pixelSize: parent.height*0.130
+            }
+            property real angle: ((index) * 2 * Math.PI / hourNumbers.count - Math.PI/2) 
+            property var extraY: [0.175, 0.044, -0.075 , 0.058 ]
+            horizontalAlignment: Text.AlignHCenter
+            opacity: displayAmbient ? 0.2 : 0.8
+            anchors.centerIn: parent
+            transform: 
+                Translate {
+                    x: Math.cos(angle) * (parent.width + width) * 0.445
+                    y: Math.sin(angle) * (parent.height + height) * 0.388 + height * extraY[index]
+                }
+            color: "#ff949494"
+            text: 3 * (index === 0 ? hourNumbers.count : index)
+        }
+    }
+	
+    Repeater {
+    model: 60
+    id: secondNumbers
+        Text {
+            z: 0
+            font.pixelSize: parent.height*0.030
+            font.family: "Michroma"
+            font.styleName: "Regular"
+            horizontalAlignment: Text.AlignHCenter
+            property real rotM: (index - 15) / 60
+            property real centerX: parent.width / 2 - width / 2
+            property real centerY: parent.height / 2 - height / 2
+			property bool south: index > 15 && index < 45
+            x: centerX + Math.cos(rotM * 2 * Math.PI) * parent.width * 0.463
+            y: centerY + Math.sin(rotM * 2 * Math.PI) * parent.width * 0.463
```

Comment:
Whenever you calculate an `x` coordinate, use `parent.width` and with the `y` coordinate, use `parent.height`.  This will help make the watchface look right even on watchfaces which are not square or round, but rectangular or elliptical (if there are ever elliptical-face watches!)